### ISSUE TITLE
java Exception in thread "main" com.nextcloud.client.di.InjectorNotFoundException

### ISSUE THREAD

**Issue by wefranke** [nextcloud/android#10862]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

1. connected divice to WLAN
2. start just installed nextcloud app
3. enter URL of own hosted nextcloud server (https://192.168.178.11/nextcloud)
4. java Exception in thread "main" com.nextcloud.client.di.InjectorNotFoundException: Injector not registered for com.owncloud.android.ui.dialog.SslUntrustedCertDialog. Have you added it to com.nextcloud.client.di.ComponentsModule?

On the server I used a Self-Signed SSL Certificate. This is working since longer time for all other Nextcloud Clients I used. 
On Android older version 3.21.2
Installation of OneCloud does not help. (because of com.**owncloud**.android.ui.dialog.SslUntrustedCertDialog)

### Expected behaviour

connection to NC server and asking for user and password

## Cause of error
```java
Exception in thread "main" com.nextcloud.client.di.InjectorNotFoundException: Injector not registered for com.owncloud.android.ui.dialog.SslUntrustedCertDialog. Have you added it to com.nextcloud.client.di.ComponentsModule?
    at com.nextcloud.client.di.FragmentInjector.onFragmentPreAttached(FragmentInjector.java:43)
    at androidx.fragment.app.FragmentLifecycleCallbacksDispatcher.dispatchOnFragmentPreAttached(FragmentLifecycleCallbacksDispatcher.java:100)
    at androidx.fragment.app.FragmentStateManager.attach(FragmentStateManager.java:463)
    at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:254)
    at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1899)
    at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1817)
    at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1760)
    at androidx.fragment.app.FragmentManager$5.run(FragmentManager.java:547)
    at android.os.Handler.handleCallback(Handler.java:938)
    at android.os.Handler.dispatchMessage(Handler.java:99)
    at android.os.Looper.loopOnce(Looper.java:226)
    at android.os.Looper.loop(Looper.java:313)
    at android.app.ActivityThread.main(ActivityThread.java:8669)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
Caused by: Exception in thread "main" java.lang.IllegalArgumentException: No injector factory bound for Class<com.owncloud.android.ui.dialog.SslUntrustedCertDialog>
    at dagger.android.DispatchingAndroidInjector.inject(DispatchingAndroidInjector.java:136)
    at dagger.android.support.AndroidSupportInjection.inject(AndroidSupportInjection.java:75)
    at dagger.android.support.AndroidSupportInjection.inject(AndroidSupportInjection.java:67)
    at com.nextcloud.client.di.FragmentInjector.onFragmentPreAttached(FragmentInjector.java:40)
    at androidx.fragment.app.FragmentLifecycleCallbacksDispatcher.dispatchOnFragmentPreAttached(FragmentLifecycleCallbacksDispatcher.java:100)
    at androidx.fragment.app.FragmentStateManager.attach(FragmentStateManager.java:463)
    at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:254)
    at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1899)
    at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1817)
    at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1760)
    at androidx.fragment.app.FragmentManager$5.run(FragmentManager.java:547)
    at android.os.Handler.handleCallback(Handler.java:938)
    at android.os.Handler.dispatchMessage(Handler.java:99)
    at android.os.Looper.loopOnce(Looper.java:226)
    at android.os.Looper.loop(Looper.java:313)
    at android.app.ActivityThread.main(ActivityThread.java:8669)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)

```

### App information
* ID: `com.nextcloud.client`
* Version: `30220090`
* Build flavor: `gplay`

### Device information
* Brand: `samsung`
* Device: `gta4lwifi`
* Model: `SM-T500`
* Id: `SP1A.210812.016`
* Product: `gta4lwifieea`

### Firmware
* SDK: `31`
* Release: `12`
* Incremental: `T500XXU3CVG4`

### Android version

12

### Device brand and model

Samsung Galaxy Tab A7

### Stock or custom OS?

Stock

### Nextcloud android app version

3.22.0

### Nextcloud server version

24.0.6

### Using a reverse proxy?

No

**Comment by AlvaroBrey** [comment]
Duplicate of #10827

**Comment by AlvaroBrey** [comment]
This is fixed in 3.22.1, please update your app
